client/components/ProductOverview/ThumbnailView.jsx

```diff
@@ -49,24 +49,26 @@ const ThumbnailView = ({ photos, handleClick, currentIndex }) => {
 
   return (
     <div className="arrowsThumbnailContainer">
-      {currentIndex !== 0 ? <AiOutlineArrowUp className="upArrow" size={arrowSize} onClick={handleArrowUp} /> : null}
+      {currentIndex !== 0 ? <AiOutlineArrowUp className="upArrow" size={arrowSize} onClick={handleArrowUp} /> : <AiOutlineArrowUp className="upArrow" size={arrowSize} onClick={handleArrowUp} style={{ visibility: 'hidden' }} />}
       <div className="thumbnailContainer">
         {photos.map((photo, index) => {
           if (index >= thumbnailStart && index < thumbnailEnd) {
             return (
-              <div style={{ backgroundImage: `url(${photo.thumbnail_url})` }} id={index === currentIndex ? 'selected' : 'unselected'} className="thumbnail" key={photo.thumbnail_url}>
-                <input
-                  className="thumbnailInput"
-                  type="button"
-                  onClick={() => handleClick(index)}
-                />
-              </div>
-
+              <input
+                className="thumbnailInput"
+                type="image"
+                alt="image from selected style set"
+                onClick={() => handleClick(index)}
+                id={index === currentIndex ? 'selected' : 'unselected'}
+                key={photo.thumbnail_url}
+                src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/89/HD_transparent_picture.png/1200px-HD_transparent_picture.png"
```

Comment:
Setting an image source and a background image still feels a little wonky here to me. I guess if it works, it's okay for now, but I wonder if there's a better way to do this. 🤔